analyzer-comments/java/need-for-speed/avoid_loops.md

```diff
@@ -0,0 +1,3 @@
+# avoid loops
+
+Consider converting the `loop` statement and returning a single expression using the `<=` operator while reusing methods already implemented in the class.

```

Comment:
I'm not sure that it makes sense to wrap the word "loop" in backticks, as it is not a keyword in Java.

Reply:
👍